# Problem statement


To find if male-female buyer ratio is simialr across all given regions

In [1]:
import pandas as pd

In [2]:
data = pd.read_csv("C:\\Users\\nilka\\OneDrive\\Desktop\\excelr\\self paced course\\data science assignment\\1.assignments\\3.hypothesis testing\\BuyerRatio.csv")

In [3]:
data

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

# Hypothesis statement

H0: the buyer ratio is similar across regions

H1: Buyer Ratio is different across regions

# TEST

# CHI SQUARE TEST

In [5]:
from scipy import stats

In [10]:
stats.chi2_contingency([data['East'],data['West'],data['North'],data['South']])

(1.5959455386610577,
 0.6603094907091882,
 3,
 array([[  42.76531299,  442.23468701],
        [ 146.81287862, 1518.18712138],
        [ 131.11756787, 1355.88243213],
        [  72.30424052,  747.69575948]]))

P value = 0.66

As p > 0.05 we reject H0.  Thus there is difference in buyer ratio across regions